resource/decker/support/plugins/menu/menu.js

```diff
@@ -516,6 +508,7 @@ class SlideMenu {
     this.menu.close_button = container.querySelector(
       "#decker-menu-close-button"
     );
+    this.menu.a11y_button = container.querySelector("#decker-menu-a11y-button");

```

Comment:
why does this button show up explicitly, when it is dynamically added in the handout plugin?

Reply:
That was a leftover from my first trys to implement the mode. Removed it.